### DISCUSSION TITLE
Forking Renovate Failed To Fork Minimal Reproduction

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#21140] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I'm trying to create a minimal reproduction in [Kurt-von-Laven/slack-templates](https://github.com/Kurt-von-Laven/slack-templates) for an unrelated issue, but the corresponding fork was never created.

### What you tried so far.

I observed that [renovate-bot/slack-templates-1](https://github.com/renovate-bot/slack-templates-1) does not exist and that when Renovate runs on the minimal reproduction, it appears that it may be using the config from [ScribeMD/slack-templates](https://github.com/ScribeMD/slack-templates). I'm guessing an internal error prevented Forking Renovate from creating the fork normally, but if my suspicion that it's reading the config from a different repository is correct, this is also a security concern.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
GitHub doesn't allow two related forks or a parent and fork to exist in the same org. You should instead *Import* the repo into a new one instead of forking it.

**Reply by rarkins** [reply]
We plan to soon explicitly disable support for forked repos in Forking Renovate to make this limitation explicit

**Reply by Kurt-von-Laven** [reply]
Doh. Great idea; thank you! I can confirm that pushing a new repo rather than forking it worked.
